autoscheduler/frontend/src/types/Meeting.ts

```diff
@@ -6,14 +6,23 @@ export enum MeetingType {
 
 export default class Meeting {
   id: number;
+
   crn: number;
+
   building: string | null;
+
   meetingDays: boolean[];
+
   startTimeHours: number;
+
   startTimeMinutes: number;
+
   endTimeHours: number;
+
   endTimeMinutes: number;
+
   meetingType: MeetingType;
+
```

Comment:
I think we want member variables to have no lines between them, but I know it's a linting rule so we need to change it to allow it.